apps/marketplace/components/Brief/Overview/BriefOverview.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import BriefOverviewSection from './BriefOverviewSection'
+
+import styles from './BriefOverview.scss'
+
+export class BriefOverview extends Component {
+  static propTypes = {
+    sections: PropTypes.array.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
+  render() {
+    const { sections, title } = this.props
+
+    return (
+      <div className="row">
+        <div className="col-md-12 col-sm-12">
```

Comment:
I believe uikit grid is like the bootstrap grid, so just `col-xs-12` here would suffice (full width on extra small, and all breakpoints above it).

Reply:
`col-xs-12`added.